docs/vgclog_examples.md

```diff
@@ -706,3 +707,895 @@ Analyzing the structure and elements of this example log output shows that this
+
+The aim of the global mark cycle to create a new record of object liveness can be seen in the remembered set statistics. The global mark cycle populates the remembered set and also attempts to completely rebuild the set for the overflowed regions. 
+
+The logs show that at the start of this STW subincrement, the remembered set count is 2,197,888 cards, the metastructure is using 94% of its total available memory, and three overflow regions need to be rebuilt.
+
+The `<gc-op>` element records that the STW subincrement runs a [mark operation](gc_overview.md#gc-mark-operation). This operation begins the process of building a record of object liveness across the heap.
+
+```xml
+<gc-op id="1157" type="mark increment" timems="122.825" contextid="1154" timestamp="2021-02-26T11:17:25.157">
+  <trace-info objectcount="7726701" scancount="7584109" scanbytes="213445656" />
+</gc-op>
+```
+
+The `<trace-info>` element records information about the marking and scanning stages of the mark increment operation. `objectcount` records the number of objects that have been marked, ready for tracing. After marking live objects, a scan is run to trace objects and references. `scancount` records the number of marked objects that have been scanned. `scanbytes` records the total memory of all marked objects that have been scanned.
+
+The mark increment operation in the example marked 7,726,701 objects and scanned 7,584,109 of these marked objects. The 7,584,109 of scanned objects take up 203.5 MB (213445656 B) of memory. The number of scanned objects is less than the number of marked objects because only objects that have children require scanning. Also, the scanning part of the marking operation might be interrupted by the garbage collector if a trigger threshold for a partial cycle or global cycle is reached during the marking operation.
+
+The STW `global mark phase` subincrement ends, as recorded by `<gc-end>`, which records a snapshot of the memory status in the heap in a similar way to `<gc-start>`. 
+
+```xml
+<gc-end id="1158" type="global mark phase" contextid="1154" durationms="123.139" usertimems="977.851" systemtimems="0.000" stalltimems="1.453" timestamp="2021-02-26T11:17:25.157" activeThreads="8">
+  <mem-info id="1159" free="1442840576" total="4294967296" percent="33">
+    <remembered-set count="3263968" freebytes="158648448" totalbytes="171704320" percent="92" regionsoverflowed="0" regionsstable="0" regionsrebuilding="133"/>
+  </mem-info>
+</gc-end>
+```
+
+Comparing the snapshot at the beginning and end of this STW `global mark phase` subincrement shows that:
+
+- The marking operation has increased the `count` value of the `<remembered-set>` by 1,066,080 cards (from 2,197,888 to 3,263,968)
+- The number of overflow regions has reduced from three to zero. This GMP increment has rebuilt all overflow regions.
+- As expected of a global mark cycle's operations, there is no change in the amount of free memory available, which is 1376 MB.
```

Comment:
true for any GMP increment - it wont reclaim memory. We are only rebuilding liveness info and RSes.